guides/common/modules/proc_configuring-repositories-proxy.adoc

```diff
@@ -43,10 +73,3 @@ For more information about modules and lifecycle streams on {RHEL} 8, see https:
 If you are installing {ProductName} as a virtual machine hosted on {oVirt}, you must also enable the *Red{nbsp}Hat Common* repository and then install {oVirt} guest agents and drivers.
```

Comment:
I'd argue this note (if it's still relevant) should be above EL 8/9 because it applies to both versions. If you place it here then it implies it's only for EL8.